Project/zepeto-modules/Assets/Zepeto Gesture Module/ZepetoScript/UIController.ts

```diff
@@ -50,22 +51,27 @@ export default class UIController extends ZepetoScriptBehaviour {
     private SetCategoryUI(category: OfficialContentType) {
         
         if (category == OfficialContentType.All) {
-            this._gestureLodaer.thumbnails.forEach((Obj) => {
+            this._gestureLoader.thumbnails.forEach((Obj) => {
                 Obj.SetActive(true);
             });
         }   else {
-            for (let i = 0; i < this._gestureLodaer.thumbnails.length; i++) {
-                const content = this._gestureLodaer.thumbnails[i].GetComponent<Thumbnail>().content;
+            for (let i = 0; i < this._gestureLoader.thumbnails.length; i++) {
+                const content = this._gestureLoader.thumbnails[i].GetComponent<Thumbnail>().content;
                 if (content.Keywords.includes(category)) {
-                    this._gestureLodaer.thumbnails[i].SetActive(true);
+                    this._gestureLoader.thumbnails[i].SetActive(true);
                 } else {
-                    this._gestureLodaer.thumbnails[i].SetActive(false);
+                    this._gestureLoader.thumbnails[i].SetActive(false);
                 }
             }
         }
     }
-    
+      
     private StopGesture() {
+        //If the coroutine stop it.
+        if(this._gestureLoader.gestureLoop && this._gestureLoader.isLooping)
```

Comment:
Is it necessary to check `isLooping` - if `gestureLoop` evaluates to be `true`?

Reply:
Ah yeah, you are right. Will change that.